tensorboard/plugins/profile/README.md

```diff
@@ -1,135 +1,89 @@
-
-*   tip_type: string
-*   link: string
-
-### 2. JSON format for the Input-pipeline Analyzer
-
-JSON for the Input-pipeline Analyzer uses three separate DataTables:
-
-*   the first DataTable contains statistics for device step time
-*   the second DataTable contains statistics for the host side
-*   the third DataTable contains recommendations to users
-
-Each DataTable is associated with some table properties and some tabular data.
-
-#### 2.1. Device Step-time DataTable
-
-The same as the one described in Section 1.2.
-
-#### 2.2 Host Side DataTable
-
-2.2.1 Properties:
-
-*   advanced_file_read_us: string (time in us spent on reading files in advance)
-*   demanded_file_read_us: string (time in us spent on reading files on demand)
-*   enqueue_us: string (time in us spent on enqueuing data to be transferred to
-    the TPU)
-*   preprocessing_us: string (time in us spent on data pre-processing)
-*   unclassified_nonenqueue_us: string (time in us spent on other components)
-
-2.2.2 Tabular data: rows are input-pipeline related tensorflow ops. column ids
-are defined below.
-
-*   opName: string
-*   count: number
-*   timeInMs: number
-*   timeInPercent: number
-*   selfTimeInMs: number
-*   selfTimeInPercent: number
-*   category: string
-
-#### 2.3 Recommendation DataTable
-
-2.3.1 Properties:
-
-*   overall: string (the overall recommendation to users)
-
-2.3.2 Tabular data: rows are recommendation details. column ids are defined
-below:
-
-*   link: string (recommendation for optimizing an individual input-processing
-    component)
+  The TensorBoard Profile Dashboard includes a suite of cloud TPU tools. These
+  tools help you understand, debug and optimize TensorFlow programs to run on
+  cloud TPUs.
+
+## Prerequisites
+  Before you can use the tools in Profile Dashboard, you must have access to
+  google cloud TPUs. You also need to capture trace information while your model
+  is running.
+
+## Overview Page
+  The Overview Page gives an overview of the performance of the workload
+  running on the TPU. It has five sections as shown in the following figure:
+
+  ![Overview Page](docs/overview-page.png)
+
+  * Performance Summary, which includes:
+
+    - The step time averaged over all steps sampled
+    - The percentage of Host time that is idle
+    - The percentage of TPU time that is idle
+    - The percentage utilization of the TPU matrix units
+
+  * Step-time Graph, which plots a graph of step time (in milliseconds) over
+  all the steps sampled.
+
+  * Top 10 TensorFlow operations on TPU, which shows the TensorFlow operations
+  executed on the TPU that consumes the majority of time.
+
+  * Run Environment, which includes:
+
+    - The number of hosts used
+    - The type of TPU used
+    - The number of TPU cores
+    - The training batch size
+    - Job information (both the build command and run command)
+    - Recommendation for Next Steps, which first reports if the workload is
```

Comment:
This should be its own top-level bullet, right?  It looks like the 5th section rather than a subsection of Run Environment.

Reply:
Nice catch!